src/app/inventory/dimCsvService.factory.ts

```diff
@@ -18,7 +18,11 @@ const FILTER_NODE_NAMES = [
   'Kinetic Damage',
   'Solar Damage',
   'Arc Damage',
-  'Void Damage'
+  'Void Damage',
+  'Default Shader',
+  'Default Ornament',
+  'Empty Mod Socket',
+  'No Projection'
```

Comment:
This of course only works in English.

Reply:
Yeah, the export feature never got i18n love.

Surprisingly, there haven't been issues about that that I've seen.